usecases/backup/handler.go

```diff
@@ -240,6 +240,48 @@ func (m *Manager) RestorationStatus(ctx context.Context, principal *models.Princ
 	return istatus.(RestoreStatus), nil
 }
 
+// OnCanCommit will be triggered when coordinator asks the node to participate
+// in a distributed backup operation
+func (m *Manager) OnCanCommit(ctx context.Context, req *Request) CanCommitResponse {
+	ret := CanCommitResponse{Method: req.Method, ID: req.ID}
+	store, err := backend(m.backends, req.Backend)
+	if err != nil {
+		ret.Err = fmt.Sprintf("no backup backend %q, did you enable the right module?", req.Backend)
+		return ret
+	}
+
+	// TODO: validate if all req.Classes exist locally
```

Comment:
What is the current behavior if at least one in `req.Classes` does not exist on the node?

Reply:
The current behaviour is return an error if any classes does't exist.